clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/provider/agent/DockerRegistryImageCachingAgent.groovy

```diff
@@ -42,19 +44,22 @@ class DockerRegistryImageCachingAgent implements CachingAgent, AccountAware {
   private String accountName
   private final int index
   private final int threadCount
+  private final Long interval
```

Comment:
Since the `cacheIntervalSeconds` field in `DockerRegistryNamedAccountCredentials` is `long`, why does this need to be `Long`?

Reply:
Good point.